overlord/servicestate/export_test.go

```diff
@@ -19,8 +19,22 @@
 
 package servicestate
 
+import (
+	"github.com/snapcore/snapd/overlord/state"
+	tomb "gopkg.in/tomb.v2"
```

Comment:
these should be laid out:

```
tomb "gopkg.in/tomb.v2"

"github.com/snapcore/snapd/overlord/state"
```

as usual

```
stdlib

3rd parties

snapcore/snapd
```

Reply:
sorry my editor automatically added the tomb dependency and I didn't notice it was added in a non-standard order like this